### PR TITLE
Adding composer install as before_deploy step to .amazeeio.yml

### PR DESCRIPTION
**Pull request by dasrecht**

### CODE REVIEW

File: .amazeeio.yml [hunk 1]
```diff
@@ -2,10 +2,14 @@ sitegroup: replaceme
 
 deploy_tasks:
   development:
+    before_deploy:
+      - composer install
     after_deploy:
       - cd web && drush -y updb --cache-clear=0
       - cd web && drush -y cr
   production:
+  before_deploy:
```
Comment (Schnitzel) [inline .amazeeio.yml]: missing spaces

**Review by Schnitzel — COMMENTED** [review]
looks good, just a small issue, when fixed, reaty to merge

File: .amazeeio.yml (hunk 1 above)
Comment (dasrecht) [reply .amazeeio.yml]: fixed :)

File: .amazeeio.yml [hunk 2]
```diff
@@ -2,15 +2,19 @@ sitegroup: replaceme
...
+    before_deploy:
+    - composer install
     after_deploy:
       - cd web && drush -y updb --cache-clear=0
       - cd web && drush -y cr
 
 shared:
   production:
     - src: files
-      dst: sites/default/files
```
Comment (dasrecht) [inline .amazeeio.yml:16]: @Schnitzel also added the `web/` here.

File: .amazeeio.yml [hunk 3]
```diff
@@ -2,15 +2,19 @@ sitegroup: replaceme
...
 deploy_tasks:
   development:
+    before_deploy:
+      - composer install
     after_deploy:
       - cd web && drush -y updb --cache-clear=0
       - cd web && drush -y cr
   production:
+    before_deploy:
+    - composer install
```
Comment (Schnitzel) [inline .amazeeio.yml]: still missing spaces ;)

File: .amazeeio.yml (hunk 2 above)
Comment (Schnitzel) [reply .amazeeio.yml:16]: 👍

File: .amazeeio.yml (hunk 3 above)
Comment (dasrecht) [reply .amazeeio.yml]: fixed for good now :)

**Pull request merged**